build-system/tasks/check-sourcemaps.js

```diff
@@ -164,7 +165,7 @@ function checkSourcemapMappings(sourcemapJson) {
     log(helpMessage);
     throwError('Found mapping for incorrect code');
   }
-  if (generatedCodeColumn != 0 || sourceCodeColumn != 0) {
+  if (generatedCodeColumn != expectedColStart || sourceCodeColumn != 0) {
```

Comment:
What is the value of `sourceCodeColumn` with your change?

Reply:
sourceCodeColumn in unchanged (which is why the check passes)